### ISSUE TITLE
Elevation Profile: Tolerance setting

### ISSUE THREAD

**Issue by Jakob-Lanstorp** [qgis/QGIS#51009]
### Feature description

The tolerance setting of the Elevation Profile is generic for all vector layers. It would be very handy, if this setting also could be set on a single vector layer through the Layer Properties > Elevation dialog. It could be a new section called override generic tolerance and a place to enter your new tolerance for that layer. 

### Additional context

As a geologisk I have build a geological profiler where borehole lithologies are added to the elevation profile through geometri generators. Now I would like to add geophysical data, that needs another tolerance , for when it is included in the profile.

**Comment by ptitjano** [comment]
This is now available in QGIS 3.42 for vector layers. Does this cover your needs?

![Image](https://github.com/user-attachments/assets/b90ed180-331e-4a31-bd8d-c67314b0e557)

**Comment by Jakob-Lanstorp** [comment]
Yes the add of a Custom Tolerance to an elevation layer property in QGIS 3.42 solves it. One remark is that having this property exposed for changing for several elevation layers at once would be very nice. I added a feature request for this in [https://github.com/qgis/QGIS/issues/60618](https://github.com/qgis/QGIS/issues/60618) on having group layers in the Elevation Layers Panel.
